src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStat.tsx

```diff
@@ -22,7 +23,7 @@ interface KeyStatColumnProps {
 
 export const KeyStatColumn = ({ children, testId }: KeyStatColumnProps) => {
   return (
-    <div className={styles.column} data-testid={testId}>
+    <div className='dfe-keyStatistics-column' data-testid={testId}>
```

Comment:
Rather than put this in `_content.scss`, couldn't we it in from `KeyStatistics.module.scss` as a `className` prop?

Reply:
Yep, don't know why I didn't do that!

Reply:
:smile: looks good now!